deployment-examples/gcp/README.md

```diff
@@ -1,21 +1,29 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  
+<!-- *generated with [DocToc](https://github.com/thlorenz/doctoc)* -->
+
+- [Continuous Deployment Guide](#continuous-deployment-guide)
+  - [Prerequisites:](#prerequisites)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
 ## Continuous Deployment Guide
 
 As a quick start, you can utilize the available bash script which is capable of deploying both modules to a 
-Google Cloud Project. It requires to have:
+public cloud provider.
+
+### Prerequisites:
 
 - [gcloud sdk](https://cloud.google.com/sdk/docs/install)
 - [docker](https://docs.docker.com/get-docker/)
 - [helm](https://helm.sh/docs/intro/install/)
 - [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
-- an existing Kubernetes cluster in [Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine/docs/quickstart)
-
-
-### Prerequisites:
-1. Configure both module specific configurations in yaml files that resides under `k8s` folders.
-
-    Refer to [`extension/docs/DeploymentGuide.md`](./../extension/docs/DeploymentGuide.md) and [`notification/docs/DeploymentGuide.md`](./../notification/docs/DeploymentGuide.md) docs.
-2. Configure the below values in [`deploy-to-gcp.sh`](./../deployment/deploy-to-gcp.sh) file.
-3. Ensure the `secrets.yaml` files ([extension](./../extension/k8s/demo/secrets.yaml) and [notification](./../notification/k8s/demo/secrets.yaml)) have encrypted using the `GCLOUD_KMS_KEY_NAME` key.
+- An existing Kubernetes cluster in [Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine/docs/quickstart)
+- Configure both module specific configurations in yaml files that resides under `k8s` folders.
+    Refer to [`extension/docs/DeploymentGuide.md`](../../extension/docs/HowToRun.md) and [`notification/docs/DeploymentGuide.md`](../../notification/docs/HowToRun.md) docs.
```

Comment:
Honestly, this is hard to get the problem in old documents, what was wrong in the old DeploymentGuide file :D even the contents are the same, why not just referencing the deployment examples in our docs?

I think it's not good to change file names as it's a breaking change for people who has the links :)

Reply:
>what was wrong in the old DeploymentGuide file 

We should not confuse the reader on executing the module in their machines vs. deploying the infrastructure for production usage. Therefore, I thought not to use "deployment" word in multiple places, rather gather all information required for deployment in one place.